definitions/npm/cache-manager_v2.x.x/flow_>=v0.28.x/cache-manager_v2.x.x.js

```diff
@@ -0,0 +1,32 @@
+declare module 'cache-manager' {
+  declare type _CacheOptions = {
+    ttl?: number,
+  }
+  declare type CacheOptions = $Shape<_CacheOptions> | Object
```

Comment:
Nope there is not.
All extra Settings get passed to the specific Store implementation. So we can't restrict it to _CacheOptions.